app/controllers/static_pages_controller.rb

```diff
@@ -47,8 +46,8 @@ def dashboard
         Room.joins(floor: :building)
             .left_outer_joins(:room_states)
             .where(buildings: { zone_id: !nil })
```

Comment:
instead of `.where(buildings: { zone_id: !nil })` write` .where.not(buildings: { zone_id: nil })`